core/src/main/resources/core/publishAndTag

```diff
@@ -16,10 +16,10 @@ fi
 set -u
 
 TRICKLE_TASKS=
-if [[ "$GITHUB_REF" == 'refs/heads/master' ]]; then
+if [[ "$GITHUB_REF" == 'refs/heads/master' ] || [ "$GITHUB_REF" == 'refs/heads/main' ]]; then
```

Comment:
Shouldn't this be

```bash
[[ "$GITHUB_REF" == 'refs/heads/master'  || "$GITHUB_REF" == 'refs/heads/main' ]]
```

?

---

Otherwise you'll be using single brackets for your conditional, [which I understand doesn't support `||`](https://tldp.org/LDP/abs/html/testconstructs.html#DBLBRACKETS).

> Using the [[ ... ]] test construct, rather than [ ... ] can prevent many logic errors in scripts. For example, the &&, ||, <, and > operators work within a [[ ]] test, despite giving an error within a [ ] construct.

(Disclaimer: I haven't done serious bash scripting in years so I might be wrong)

Reply:
@jsantos17 Thanks! I think you are right.